courses/machine_learning/deepdive2/structured/solutions/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,1593 @@
+{
```

Comment:
<p>if I remember correctly, there was a preference to use a standard format for setting PROJECT and BUCKET for labs/solutions involving CAIP. The intent was to be able to run a 'setup' notebook that does find/replace on particular string for the PROJECT. Not sure how widely is has been observed within the existing labs but might as well adhere here.</p><p> </p><p>According to this doc,</p><p><a href="https://github.com/GoogleCloudPlatform/training-data-analyst/tree/master/courses/machine_learning/deepdive2/_instructors" target="_blank">https://github.com/GoogleCloudPlatform/training-data-analyst/tree/master/courses/machine_learning/deepdive2/_instructors</a></p><p>it should have this format:</p><p> </p><p>%%bash</p><p>import os</p><p> </p><p>PROJECT = "your-gcp-project-here" # REPLACE WITH YOUR PROJECT NAME</p><p>REGION = "us-central1" # REPLACE WITH YOUR BUCKET REGION e.g. us-central1</p><p> </p><p># Do not change these</p><p>os.environ["PROJECT"] = PROJECT</p><p>os.environ["REGION"] = REGION</p><p>os.environ["BUCKET"] = PROJECT # DEFAULT BUCKET WILL BE PROJECT ID</p><p> </p><p> </p><p>if changed, change in lab notebook as well.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>os.environ["PROJECT"] = PROJECT is redundant from the export PROJECT=$(gcloud config list project --format "value(core.project)") which I saw in previous labs and thought it was good so copied it into mine too hehe. The rest is basically the same as what I got.</p>